services/ml/common.cc

```diff
@@ -27,6 +27,10 @@ uint32_t GetRequiredSize(int32_t type,
     return product(dimensions) * sizeof(int32_t);
   } else if (type == mojom::TENSOR_QUANT8_ASYMM) {
     return product(dimensions) * sizeof(int8_t);
```

Comment:
Should be `uint8_t`. This is not introduced in this PR, but it would be good to fix that.